spec/factories/allocation_decisions.rb

```diff
@@ -22,8 +22,8 @@
       association :hearing, factory: :realistic_hearing
       association :offence, factory: :realistic_offence
       motReasonId { SecureRandom.uuid }
-      motReasonDescription { Faker::Offence.mode_of_trial_reason }
-      motReasonCode { Faker::Alphanumeric.alphanumeric(number: 4) }
+      motReasonDescription { "Indictable only offence" }
```

Comment:
The realistic factory should be random if possible. 

If anything, we should change the normal factory to use hard coded values.

Reply:
Realistic factory changed because single description needs to be matched to specific code

Reply:
Yes, understood. If it is possible to get a random one that then autopopulates the code for the random description. I would look into whether `self.motReasonCode = AllocationDecision::DESCRIPTIONS[motReasonDescription]` worked.

That was what I meant by if possible. If not, it's not a big deal.

Could we fix the factory above though, to hardcode the two attributes in the same way?